R/create_service_use_lookup.R

```diff
@@ -0,0 +1,330 @@
+#' Title Create the Service Use Cohort lookup file
+#'
+#' @param data A data frame
+#' @param year The financial year
+#' @param write_to_disk Default [TRUE]
```

Comment:
Square brackets are for links 😉
```suggestion
#' @param write_to_disk Default `TRUE`
```

Reply:
Done, thanks!